homework02/rudolf/string.cpp

```diff
@@ -0,0 +1,73 @@
+#include "string.h"
```

Comment:
General comment: good implementation! There is one recurrent bug around size / capacity though, they are being mixed a bit everywhere